bigquery/google/cloud/bigquery/job.py

```diff
@@ -2266,6 +2266,29 @@ def cache_hit(self):
         """
         return self._job_statistics().get('cacheHit')
 
+    @property
+    def ddl_operation_performed(self):
+        """Union[str, None]: Return the DDL operation performed.
```

Comment:
TIL there's an `Optional[X]` https://docs.python.org/3/library/typing.html#typing.Optional which is "equivalent to `Union[X, None]`. Let's use that here.